monitor/test_result.py

```diff
@@ -138,17 +138,20 @@ def __init__(
         self.table_full_name = '.'.join(table_full_name_parts).lower()
         self.detected_at = None
         if detected_at:
-            detected_at_utc = datetime.fromisoformat(detected_at)
-            self.detected_at_utc = detected_at_utc.strftime('%Y-%m-%d %H:%M:%S')
-            self.detected_at = convert_utc_time_to_local_time(detected_at_utc).strftime('%Y-%m-%d %H:%M:%S')
-        self.owners = try_load_json(owners) if owners else ''
+            try:
+                detected_at_utc = datetime.fromisoformat(detected_at)
+                self.detected_at_utc = detected_at_utc.strftime('%Y-%m-%d %H:%M:%S')
+                self.detected_at = convert_utc_time_to_local_time(detected_at_utc).strftime('%Y-%m-%d %H:%M:%S')
+            except (ValueError, TypeError):
+                logger.error(f'Failed to parse "detect_at" field.')
+        self.owners = try_load_json(owners) or ''
```

Comment:
I think that maybe we should do this owners and tags parsing inside 'to_slack_message' so it won't impact the report? I think the repot expects a list and here in the alerts we convert it to string